repose-aggregator/tests/functional-test-framework/src/main/groovy/org/openrepose/framework/test/ReposeValveLauncher.groovy

```diff
@@ -47,6 +47,8 @@ class ReposeValveLauncher extends ReposeLauncher {
     def debugPort = null
     def classPaths = []
     def additionalEnvironment = [:]
+    def xmx = "1024M"
+    def xms = "512M"
```

Comment:
Hmm, I think we should move these to be default values on the `start` method? Something like,

```groovy
start(boolean killOthersBeforeStarting, boolean waitOnJmxAfterStarting,
      String nodeId, String xmx = "1024M", String xms = "512M") {
```